google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/tag_template_constants.py

```diff
@@ -0,0 +1,63 @@
+from google.cloud import datacatalog_v1beta1
+
+
+class TagTemplateConstants:
+    """
+    Constants from this class are used both by
+    prepare.DataCatalogTagTemplateFactory and by
+    respective tests. This class is intended to be the
+    source of truth regarding which fields are expected to exist
+    in the tag templates.
+    """
+
+    DISPLAY_NAME_IDX = 0
+    FIELD_TYPE_IDX = 1
+    IS_REQUIRED_IDX = 2
+
+    @staticmethod
+    def get_fields_dict_for_cluster_tag_template():
+        fields_dict = {
+            'num_brokers':
+                ('Number of brokers',
```

Comment:
Objects with methods instead of tuples with indexes, perhaps?

Reply:
Great idea, thanks! Exactly what I was looking for. Pls see the changes